artemis/src/main/java/com/artemis/Inject.java

```diff
@@ -0,0 +1,16 @@
+package com.artemis;
+
+/**
+ *
+ */
+public interface Inject {
+
+    void initialize(World world, WorldConfiguration config);
+
+    void update();
+
+    void inject(Object target) throws RuntimeException;
+
+    boolean injectionSupported(Object target);
```

Comment:
Ah, I just meant the name - `boolean supportsInjection(Object)` sounds better, no? (If no, then keep the name)